pkgs/development/libraries/ftxui/default.nix

```diff
@@ -1,51 +1,45 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, fetchpatch
 , cmake
 , doxygen
+, gbenchmark
 , graphviz
+, gtest
 }:
 
 stdenv.mkDerivation rec {
   pname = "ftxui";
-  version = "3.0.0";
+  version = "4.1.0";
 
   src = fetchFromGitHub {
     owner = "ArthurSonzogni";
     repo = "ftxui";
     rev = "v${version}";
-    sha256 = "sha256-2pCk4drYIprUKcjnrlX6WzPted7MUAp973EmAQX3RIE=";
+    sha256 = "sha256-6uPlJXuWcTUnpk+xe6EWDYvDLsfy7hGkEMO/2j3Dz0o=";
   };
 
-  patches = [
-    # Can be removed once https://github.com/ArthurSonzogni/FTXUI/pull/403 hits a stable release
-    (fetchpatch {
-      name = "fix-postevent-segfault.patch";
-      url = "https://github.com/ArthurSonzogni/FTXUI/commit/f9256fa132e9d3c50ef1e1eafe2774160b38e063.patch";
-      sha256 = "sha256-0040/gJcCXzL92FQLhZ2dNMJhNqXXD+UHFv4Koc07K0=";
-    })
-  ];
-
   nativeBuildInputs = [
     cmake
     doxygen
+    gbenchmark
     graphviz
+    gtest
```

Comment:
I think `gtest` and `gbenchmark` should be in nativeCheckInputs
```nix
nativeCheckInputs = [
  gbenchmark
  gtest
];
```

Reply:
done.